File: BBIS.Application/DTOs/UserAccount/CreateUserAccountDto.cs
namespace BBIS.Application.DTOs.UserAccount
{
    public class CreateUserAccountDto: UserAccountBaseDto
    {
        public string Password { get; set; }
        public List<Guid> RoleIds { get; set; }
        public List<Guid> UserModuleIds { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/UserAccounts/CreateUserAccountDto.ts
import { IUserAccountBaseDto } from "./IUserAccountBaseDto";

export interface ICreateUserAccountDto extends IUserAccountBaseDto {
    Password: string | null;
    RoleIds: Array<Guid>;
    UserModuleIds: Array<Guid>;
}

export default class CreatUserAccountDto implements ICreateUserAccountDto {
    Username = null;
    Firstname = null;
    Lastname = null;
    EmailAddress = null;
    Password = null;
    RoleIds = [];
    UserModuleIds = [];
}